Library/Formula/apous.rb

```diff
@@ -0,0 +1,23 @@
+class Apous < Formula
+  desc "Apous is a simple tool that allows for easier authoring of Swift scripts."
+  homepage "https://github.com/owensd/apous"
+  url "https://github.com/owensd/apous/archive/v0.2.1.tar.gz"
+  sha256 "d2a283bd328c7dae117de6078448e87a3f3b960876608dd238f5b87f698a10bc"
+
+  depends_on :xcode => ["7.0", :build]
```

Comment:
Sorry, to be clear. You still need the xcodebuild command line utilities for it to work. 

Sent from my iPhone

> On Jul 7, 2015, at 6:13 PM, Dominyk Tiller <notifications@github.com> wrote:
> 
> In Library/Formula/apous.rb:
> 
> > @@ -0,0 +1,23 @@
> > +class Apous < Formula
> > +  desc "Apous is a simple tool that allows for easier authoring of Swift scripts."
> > +  homepage "https://github.com/owensd/apous"
> > +  url "https://github.com/owensd/apous/archive/v0.2.1.tar.gz"
> > +  sha256 "d2a283bd328c7dae117de6078448e87a3f3b960876608dd238f5b87f698a10bc"
> > +
> > +  depends_on :xcode => ["7.0", :build]
> Just to double check: You can remove xcode from your system after installing this and it'll continue to work as expected?
> 
> —
> Reply to this email directly or view it on GitHub.
>